task/pytest/0.2/pytest.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: pytest
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/categories: Testing
+    tekton.dev/pipelines.minVersion: "0.17.0"
+    tekton.dev/tags: python, pytest
+    tekton.dev/displayName: pytest
+    tekton.dev/platforms: "linux/amd64,linux/s390x,linux/ppc64le"
+spec:
+  description: >-
+    This task will run pytest on the provided input.
+
+  workspaces:
+    - name: source
+    - description: The workspace consisting of the custom pip settings.
+      name: pip-conf
+      optional: true
+  params:
+    - name: PYTHON_IMAGE
+      description: The used Python image
+      type: string
+      default: "docker.io/python:latest"
```

Comment:
let's avoid using the latest tag, let's tag this with a particular python version so that there is proper compatibility with the task. If somebody uses a higher version than defined in this task and faces any issue they can probably report it. WDYT?

Reply:
Totally agreed. In 0.1 it was latest, but at our corp we were using the image field to get a stable task as well (and be able to let dependency update tools monitor it better as well). Changed it to `3.11.1-bullseye`